specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-08-01/managedClusters.json

```diff
@@ -3868,6 +3868,71 @@
       "title": "A time range.",
       "description": "For example, between 2021-05-25T13:00:00Z and 2021-05-25T14:00:00Z."
     },
+    "ScheduledEvent": {
```

Comment:
are we going to add this to stable version?

Reply:
I think we should do that after preview